orchestrator/db/sorting/product_block.py

```diff
@@ -0,0 +1,11 @@
+from sqlalchemy.inspection import inspect
+
+from orchestrator.db import ProductBlockTable
+from orchestrator.db.sorting.sorting import generic_column_sort, generic_sort
+from orchestrator.utils.helpers import to_camel
+
+PRODUCT_BLOCK_SORT_FUNCTIONS_BY_COLUMN = {
+    to_camel(key): generic_column_sort(value) for [key, value] in inspect(ProductBlockTable).columns.items()
```

Comment:
List syntax is not needed here

```suggestion
    to_camel(key): generic_column_sort(value) for key, value in inspect(ProductBlockTable).columns.items()
```